Google.Api.Generator.Tests/ProtoTests/ResourceNames/Testing.ResourceNames/ResourceNamesResourceNames.g.cs

```diff
@@ -509,7 +509,7 @@ public enum ResourceNameType
             new WildcardMultiPatternMultipleName(ResourceNameType.ItemA, itemAId: gax::GaxPreconditions.CheckNotNullOrEmpty(itemAId, nameof(itemAId)));
 
         /// <summary>
-        /// Creates a <see cref="WildcardMultiPatternMultipleName"/> with the pattern <c>items_b/{item_b_id}</c>.
+        /// Creates a <see cref="WildcardMultiPatternMultipleName"/> with the pattern <c>items_b/{item_b_id=*}</c>.
```

Comment:
I suspect it would actually be *nicer* to omit the `=*` and `=**` parts from the documentation, but we definitely don't need to do that now. (It's *almost* an implementation detail.)

Reply:
ack